Sources/Models/Sender.swift

```diff
@@ -25,7 +25,8 @@
 import Foundation
 
 /// An object that groups the metadata of a messages sender.
-public struct Sender {
+@available(*, deprecated: 2.0, message: "`Sender` has been replaced with the `SenderType` protocol in 2.0")
```

Comment:
This needs to become 3.0. Let's add a CHANGELOG entry please 👍 

### Changed
- **API Breaking** Deprecated the `Sender` struct in favor of the `SenderType` protocol.